modular_skyrat/modules/customization/game/objects/items/plushes.dm

```diff
@@ -9,4 +9,91 @@
 	attack_verb_continuous = list("pecks", "peeps")
 	attack_verb_simple = list("peck", "peep")
 	squeak_override = list('modular_skyrat/modules/emotes/sound/voice/peep_once.ogg' = 1)
-	
+
+	/obj/item/toy/plush/deer
```

Comment:
There's space here. In fact, there are spaces everywhere. Indent with tabs.

Reply:
Aa nvm actually that's just github looking weird

Reply:
yup , just saw it too my bad

Reply:
well one of the failed checks was cuz of a bad indent so that was kinda true. it should be good now